packages/aws-client/tests/test-S3.js

```diff
@@ -169,6 +163,35 @@ test('getS3Object() will retry if the requested key does not exist', async (t) =
   t.is(response.Body.toString(), 'asdf');
 });
 
+test('getObject() returns an existing S3 object', async (t) => {
+  const { Bucket } = t.context;
+  const { Key } = await stageTestObjectToLocalStack(Bucket, 'asdf');
+
+  const response = await getObject(awsServices.s3(), { Bucket, Key });
+
+  t.is(response.Body.toString(), 'asdf');
+});
+
+test('getObject() throws an exception if the requested bucket does not exist', async (t) => {
+  const Bucket = randomString();
+
+  const promisedObject = getObject(awsServices.s3(), { Bucket, Key: 'asdf' });
+
+  const error = await t.throwsAsync(pTimeout(promisedObject, 5000));
```

Comment:
Is there any reason to make this wait 5 seconds? Why not 500 milliseconds to make the tests run a little faster

Reply:
Could do, I suppose, but I was just going off of existing test code.

Reply:
Let's not worry about it changing it for now